src/Spout/Writer/XLSX/Helper/StyleHelper.php

```diff
@@ -92,18 +92,17 @@ protected function registerBorder($style)
         $styleId = $style->getId();
 
         if ($style->shouldApplyBorder()) {
-
             $border = $style->getBorder();
             $serializedBorder = serialize($border);
 
-            if (!isset($this->registeredBorders[$serializedBorder])) {
-
+            if (isset($this->registeredBorders[$serializedBorder])) {
```

Comment:
can you extract the content of the `if` into a variable? easier to read

`$isBorderAlreadyRegistered = isset($this->registeredBorders[$serializedBorder])`